### PR TITLE
New Serverless Pattern - Email Notification with S3 pre-signed URL using SAM

### PR DESCRIPTION
**Pull request by biswanathmukherjee**
*Issue #, if available:*

*Description of changes:*

This sample project demonstrates how to send an email with pre-signed Amazon S3 bucket URL whenever a file is uploaded into the bucket using SAM.
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: s3-lambda-ses-sam/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Send email with pre-signed URL whenever a file is uploaded into S3 bucket",
```
Comment (pputhran) [inline s3-lambda-ses-sam/example-pattern.json:2]: Can the title be shortened?

**Review by pputhran — CHANGES_REQUESTED** [review]
Just a minor comment about the title. rest looks good.

File: s3-lambda-ses-sam/example-pattern.json (hunk 1 above)
Comment (biswanathmukherjee) [reply s3-lambda-ses-sam/example-pattern.json]: I have shorted the title further. Please let me know if any further changes required.

**Comment by biswanathmukherjee** [comment]
Hi @pputhran , I have updated the PR as per your review comment. Can you please re-review it?

**Review by pputhran — APPROVED** [review]

**Pull request merged**